lib/ZnapZend/ZFS.pm

```diff
@@ -122,6 +123,30 @@ sub snapshotExists {
     return grep { $snapshot eq $_ } @snapshots;
 }
 
+sub createDataSet {
+    my $self = shift;
+    my $dataSet = shift;
+    my $remote;
+
+    #just in case if someone aks to check '';
+    return 0 if !$dataSet;
+
+    ($remote, $dataSet) = $splitHostDataSet->($dataSet);
+    my @ssh = $self->$buildRemote($remote,
+        [@{$self->priv}, qw(zfs create), $dataSet]);
+
+    print STDERR '# ' . join(' ', @ssh) . "\n" if $self->debug;
+
```

Comment:
Hmm, do you want this to be debug instead of info? I've added the info below this reply.. :smile: